project_one.html

```diff
@@ -0,0 +1,199 @@
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> BACK END DEV </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> 2015 </span>
+          </div>
+          <p class="para">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="links">
+            <ul>
+              <li class="link">html</li>
+              <li class="link">css</li>
+              <li class="link">javaScript</li>
+            </ul>
+          </div>
+          <a href="#" class="pro"> See Project </a>
+        </section>
+      </div>
+
+      <br />
+      <br />
+      <div class="card">
+        <section class="snap">
+          <img src="./images/Snap3.png" alt="img" />
+        </section>
+        <section class="main">
+          <h1>Tonic</h1>
+          <div class="mid">
+            <span class="can"> CANOPY </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> BACK END DEV </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> 2015 </span>
+          </div>
+          <p class="para">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="links">
+            <ul>
+              <li class="link">html</li>
+              <li class="link">css</li>
+              <li class="link">javaScript</li>
+            </ul>
+          </div>
+          <a href="#" class="pro"> See Project </a>
+        </section>
+      </div>
+
+      <br />
+      <br />
+
+      <div class="card">
+        <section class="snap">
+          <img src="./images/Snap4.png" alt="img" />
+        </section>
+        <section class="main">
+          <h1>
+            Multi-Post <br />
+            Stories
+          </h1>
+          <div class="mid">
+            <span class="can"> CANOPY </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> BACK END DEV </span>
+            <img class="dot" src="./images/Counter.png" alt="" />
+            <span class="side"> 2015 </span>
+          </div>
+          <p class="para">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="links">
+            <ul>
+              <li class="link">html</li>
+              <li class="link">css</li>
+              <li class="link">javaScript</li>
+            </ul>
+          </div>
+          <a href="#" class="pro"> See Project </a>
+        </section>
+      </div>
+      <br />
+      <br />
+
+      <section class="about">
+        <h1 class="about_h">
+          About <br />
+          Myself
+        </h1>
+        <p class="para">
```

Comment:
- [ ] Please make sure that the background here matches the one in the Figma template
![image](https://user-images.githubusercontent.com/74744787/159127455-b739fca7-8a33-48c0-9c90-0adb18333243.png)

![image](https://user-images.githubusercontent.com/74744787/159127443-2feed0a2-0fcb-468d-9a5c-f8942c196fcc.png)